### PR TITLE
Fix/387 dist

### PR DESCRIPTION
**Pull request by tanx**
Same as @evilaliv3's PR only with dist instead of src for node unit tests.

### CODE REVIEW

**Comment by evilaliv3** [comment]
@tanx: i just retested a cleak pull + build + grunt test on my pc and it works! ok for mergin this patch if it passes travis!

my only doubt on the question you did here, but i let it to you as i've not experience on this: https://github.com/openpgpjs/openpgpjs/pull/389#discussion-diff-50606116

File: Gruntfile.js [hunk 1]
```diff
@@ -48,9 +48,7 @@ module.exports = function(grunt) {
           'test/lib/unittests-bundle.js': [ './test/unittests.js' ]
         },
         options: {
-          browserifyOptions: {
-            external: [ 'openpgp', 'crypto', 'node-localstorage']
-          }
+          external: ['crypto', 'node-localstorage', 'openpgp', '../../../dist/openpgp', '../../dist/openpgp']
```
Comment (tanx) [inline Gruntfile.js]: @evilaliv3 to answer your question... this is how things should be excluded just to be sure the browserified unittest.js build does not contain any src files in any of the unit tests. E.g. `test/crypto/cipher/aes.js` is one level deeper than `test/crypto/crypto.js`.

Comment (evilaliv3) [inline Gruntfile.js]: clear! thanks so much for the clarification

**Comment by tanx** [comment]
Ok. The travis build works! Merging...

**Pull request merged**